extensions/kubernetes-client/deployment/src/main/java/io/quarkus/kubernetes/client/deployment/DevServicesKubernetesProcessor.java

```diff
@@ -343,7 +343,7 @@ public Map<String, String> getKubeconfig() {
                 return getKubernetesClientConfigFromKubeConfig(
                         KubeConfigUtils.parseKubeConfig(KubeConfigUtils.replaceServerInKubeconfig(containerAddress.getUrl(),
                                 getFileContentFromContainer(KIND_KUBECONFIG))));
-            } else if (image.contains("k8s.gcr.io/kube-apiserver")) {
+            } else if (image.contains("registry.k8s.io/kube-apiserver")) {
```

Comment:
Shouldn't we support both for a while?

Reply:
did not think about that, I'll have a look tomorrow, to tired right now.

Reply:
I adjusted it and force pushed!